packages/manager/modules/pci/src/projects/project/storages/containers/containers.service.js

```diff
@@ -584,9 +584,10 @@ export default class PciStoragesContainersService {
     s3StorageType,
   ) {
     const region = containerRegion || OPENIO_DEFAULT_REGION;
+    const prefix = /\.[a-z]?html?$/i.test(objectKey) ? '/engine/apiv6' : '';
     return this.$http
       .delete(
-        `/cloud/project/${projectId}/region/${region}/${s3StorageType}/${containerId}/object/${encodeURIComponent(
+        `${prefix}/cloud/project/${projectId}/region/${region}/${s3StorageType}/${containerId}/object/${encodeURIComponent(
```

Comment:
Do you try with `{serviceType: 'apiv6'},` instead ?

Seems weird it's not managed by default with the global http config/interceptors

Reply:
Yup I've tried
Nothing related to this is in the interceptors as well
Like if some magic happens when requesting html files with the delete option :man_shrugging: